app/controllers/translations_controller.rb

```diff
@@ -4,10 +4,12 @@ class TranslationsController < ApplicationController
   before_action :authenticate_user!, except: [:index]
 
   def self.compute_bad_i18n_keys
-    english = locale_to_translation('en')
+    base_locales = Locales::AVAILABLE.transform_keys(&:to_s).transform_values { |attrs| attrs[:base_locale] || 'en' }
```

Comment:
```suggestion
    base_locales = Locales::AVAILABLE.transform_keys(&:to_s).transform_values { it[:base_locale] || 'en' }
```

Reply:
For context: `it` can be used as the default parameter name for lambdas or blocks that have only one argument, since Ruby 3.4 (which we are conveniently already using :wink:)